### PR TITLE
Create Readme.md

### PR DESCRIPTION
**Pull request by snehal-vahule**
Added file for dependency scanning using OWASP with SonarQube.

### CODE REVIEW

File: tools/dependency-check-OWASP/Readme.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+Sonarqube offers a plugin called Dependency-Check, a utility that attempts to detect publicly disclosed vulnerabilities contained within project dependencies. It does this by determining if there is a Common Platform Enumeration (CPE) identifier for a given dependency. If found, it will generate a report linking to the associated CVE entries.
+
+Dependency-Check supports the identification of project dependencies in a number of different languages including Java, .NET, Node.js, Ruby, and Python.
+
+
+## Note
+
+This SonarQube plugin does not perform analysis, rather, it reads existing Dependency-Check reports. Use one of the other available methods to scan project dependencies and generate the necessary JSON report which can then be consumed by this plugin. Refer to the [Dependency-Check project](https://github.com/jeremylong/DependencyCheck) for relevant [documentation](https://jeremylong.github.io/DependencyCheck/).
+
+Is it veru quick and easy to configure depenedency scanning in your azure devops pipeline.
```
Comment (iw-ezequiel) [inline tools/dependency-check-OWASP/Readme.md:15]: Typo :)

File: tools/dependency-check-OWASP/Readme.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+## Dependency scanning with SonarQube
+SonarQube empowers all developers to write cleaner and safer code. SonarQube provides clear remediation guidance for developers to understand and fix issues and for teams overall to deliver better, safer software.
+
+<!--Security Vulnerabilities require immediate action. SonarQube provides detailed issue descriptions and code highlights that explain why your code is at risk.-->
+
```
Comment (iw-ezequiel) [inline tools/dependency-check-OWASP/Readme.md:5]: Maybe get straight to the point here, as generic descriptions of Sonarqube are also elsewhere in the framework?

File: tools/dependency-check-OWASP/Readme.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+## Note
+
+This SonarQube plugin does not perform analysis, rather, it reads existing Dependency-Check reports. Use one of the other available methods to scan project dependencies and generate the necessary JSON report which can then be consumed by this plugin. Refer to the [Dependency-Check project](https://github.com/jeremylong/DependencyCheck) for relevant [documentation](https://jeremylong.github.io/DependencyCheck/).
+
+Is it veru quick and easy to configure depenedency scanning in your azure devops pipeline.
+
+- Install Dependency-Check plugin on SonarQube.
+- In your azure build pipeline add SonarQube prepare task
```
Comment (iw-ezequiel) [inline tools/dependency-check-OWASP/Readme.md:18]: Would be worth creating a section for Azure - we'll probably need similar for Jenkins/Github actions/Gitlab CI?

**Comment by andyblundell** [comment]
Closing as now out of date

**Pull request closed without merging**